### PR TITLE
Approver permissions for 1.28 Release

### PR DESCRIPTION
**Pull request by Rishit-dagli**
As the 1.28 Release Docs lead, I would like to add myself to the approver list for dev-1.28 branch.

/cc @gracenng 
/cc @leonardpahlke

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -37,6 +37,7 @@ aliases:
     - nate-double-u
     - onlydole
     - reylejano
+    - Rishit-dagli
```
Comment (leonardpahlke) [inline OWNERS_ALIASES:40]:
```suggestion
    - Rishit-dagli # 1.28 Release Team Docs Lead
```

**Review by leonardpahlke — COMMENTED** [review]
I think we update owners each cycle, right?
Could you remove @mickeyboxell as previous docs lead from the list? 
Thanks

**Comment by Rishit-dagli** [comment]
@leonardpahlke, Updated according to comments and squashed, could you please take a look again?

**Review by leonardpahlke — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**